src/black/lines.py

```diff
@@ -200,9 +200,9 @@ def is_triple_quoted_string(self) -> bool:
         value = self.leaves[0].value
         if value.startswith(('"""', "'''")):
             return True
-        if Preview.accept_raw_docstrings in self.mode and value.startswith(
-            ("r'''", 'r"""', "R'''", 'R"""')
```

Comment:
hm, here the old formatting seems better

Reply:
Perhaps the rule should be that if the inner collection fits on a single line, we keep the parentheses with the inner collection.

Reply:
Agreed. Updated so it no longer hugs if the pre-hugged version fits on a single line.